web_auto_refresh/static/src/js/web_auto_refresh.js

```diff
@@ -0,0 +1,71 @@
+/* -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2013 Fisher Yu
+#    All Rights Reserved
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+############################################################################*/
```

Comment:
I am not sure how strictly OCA adheres to these policies, but I think the license/copyright header to use here should be the same as in the [template OCA module](https://github.com/OCA/maintainer-tools/blob/master/template/module/static/tests/js/module_name.js#L2).

Otherwise am ready to approve this module!

Reply:
i've reworked header to match the oca template module